feature/main/src/main/java/io/github/droidkaigi/confsched2023/main/MainScreen.kt

```diff
@@ -84,40 +85,45 @@ fun MainScreen(
     )
 }
 
+sealed class IconRepresentation {
+    data class Vector(val imageVector: ImageVector) : IconRepresentation()
+    data class Drawable(@DrawableRes val drawableId: Int) : IconRepresentation()
+}
+
 enum class MainScreenTab(
     val icon: ImageVector,
-    val selectedIcon: ImageVector,
+    val selectedIcon: IconRepresentation,
     val label: String,
     val contentDescription: String,
     val testTag: String = "mainScreenTab:$label",
 ) {
     Timetable(
         icon = Icons.Outlined.CalendarMonth,
-        selectedIcon = Icons.Filled.CalendarMonth,
+        selectedIcon = IconRepresentation.Vector(Icons.Filled.CalendarMonth),
         label = MainStrings.Timetable.asString(),
         contentDescription = MainStrings.Timetable.asString(),
     ),
     FloorMap(
         icon = Icons.Outlined.Map,
-        selectedIcon = Icons.Filled.Map,
+        selectedIcon = IconRepresentation.Drawable(drawableId = R.drawable.icon_map_fill),
```

Comment:
Since Icons.Filled.Map did not align with the Figma design, I added the map_fill SVG to display.